### ISSUE TITLE
Pressable Cursor is not pointer

### ISSUE THREAD

**Issue by hosseinmd** [necolas/react-native-web#1764]
**The problem**
The Pressable component cursor isn't pointer

**How to reproduce**
Simplified test case: https://codesandbox.io/s/pedantic-wilson-s34js?file=/src/App.js:0-344

Steps to reproduce:
1.
2.
3.

**Expected behavior**
Expected to cursor be pointer

**Environment (include versions). Did this work in previous versions?**

* React Native for Web (version): 0.13.16
* React (version): 16.13.1
* Browser: chrome

**Comment by necolas** [comment]
Fixed in 0.14.1

**Comment by devlopersabbir** [comment]
### I think it's not possible to make a cursor pointer on the Android and ISO platforms.
> For example
- In the Android mode how we can hover LOL😂
- Also same for the ISO.
**However** we can use Pressable components to feel the cursor pointer.
Since I love to use [Native-base](https://docs.nativebase.io) `UI Library` so in the down below have an example to use the Pressable component.
```js
<Pressable onPress={() => console.log("I'm Pressed")} rounded="8" overflow="hidden" borderWidth="1" borderColor="coolGray.300" maxW="96" shadow="3" bg="coolGray.100" p="5">
        <Box>
          <HStack alignItems="center">
            <Badge colorScheme="darkBlue" _text={{
            color: "white"
          }} variant="solid" rounded="4">
              Business
            </Badge>
            <Spacer />
            <Text fontSize={10} color="coolGray.800">
              1 month ago
            </Text>
          </HStack>
          <Text color="coolGray.800" mt="3" fontWeight="medium" fontSize="xl">
            Marketing License
          </Text>
          <Text mt="2" fontSize="sm" color="coolGray.700">
            Unlock powerfull time-saving tools for creating email delivery and
            collecting marketing data
          </Text>
          <Flex>
            <Text mt="2" fontSize={12} fontWeight="medium" color="darkBlue.600">
              Read More
            </Text>
          </Flex>
        </Box>
      </Pressable>
```
**Thanks**
